### PR TITLE
Add GUS (Agile Accelerator) connector 

### PR DESCRIPTION
**Pull request by Roma-Kyrnis**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

> Fix [#184 ]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

**Review by cjwooo — CHANGES_REQUESTED** [review]
Please slack me your config for source acceptance testing (usual procedure).

**Comment by cjwooo** [comment]
@Roma-Kyrnis can you please address the comments, resolve merge conflict, and send me the config for source acceptance tests?

File: destinations/faros-destination/src/converters/agileaccelerator/works.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        0,
+        AgileacceleratorCommon.MAX_DESCRIPTION_LENGTH
+      ),
+      url: work.baseUrl.concat(work.attributes.url),
+      type: AgileacceleratorCommon.toType(work.agf__Type__c),
+      priority: work.agf__Priority__c,
+      status: AgileacceleratorCommon.toStatus(work.agf__Status__c),
+      points: work.agf__Story_Points__c,
+      // TODO:  "Stores additional fields not explicitly tracked by the model"
+      // additionalFields: [tms_TaskField]
```
[the remark this answers is not in the document]
Comment (Roma-Kyrnis) [reply destinations/faros-destination/src/converters/agileaccelerator/works.ts:111]: Which fields do I need to put on this additionalFields?

[the remark this answers is not in the document]
Comment (Roma-Kyrnis) [reply destinations/faros-destination/src/converters/agileaccelerator/works.ts:111]: Here is all fields:

[the remark this answers is not in the document]
Comment (Roma-Kyrnis) [reply destinations/faros-destination/src/converters/agileaccelerator/works.ts:111]: <details><summary>CLICK ME</summary>

    ``` JSON
      {
    "attributes": {
        "type": "agf__ADM_Work__c",
        "url": "/services/data/v53.0/sobjects/agf__ADM_Work__c/a1F7Q000000TRimUAG"
    },
    "Id": "a1F7Q000000TRimUAG",
    "OwnerId": "0057Q000000x0FIQAY",
    "IsDeleted": false,
    "Name": "W-000000",
    "RecordTypeId": "0127Q000000omkfQAA",
    "CreatedDate": "2021-12-06T16:15:02.000+0000",
    "CreatedById": "0057Q000000x0FIQAY",
    "LastModifiedDate": "2021-12-06T16:15:02.000+0000",
    "LastModifiedById": "0057Q000000x0FIQAY",
    "SystemModstamp": "2021-12-06T16:15:06.000+0000",
    "LastActivityDate": null,
    "LastViewedDate": "2021-12-26T18:46:16.000+0000",
    "LastReferencedDate": "2021-12-26T18:46:16.000+0000",
    "agf__Additional_Details__c": null,
    "agf__Age_With_Scrum_Team_When_Closed__c": 0.0,
    "agf__Age_With_Scrum_Team__c": 37.0,
    "agf__Age__c": 51.0,
    "agf__Apex_Hammer_Compile_Failure__c": false,
    "agf__Assigned_On__c": null,
    "agf__Assignee__c": null,
    "agf__Assignees__c": null,
    "agf__Attributes__c": null,
    "agf__Auto_Build__c": null,
    "agf__Backburner_Rank__c": null,
    "agf__Board_Column_Rank__c": null,
    "agf__Board_Column__c": null,
    "agf__Branch__c": null,
    "agf__Budget_ID__c": null,
    "agf__Bug_Number__c": null,
    "agf__CS_Contact__c": null,
    "agf__Capex_Enabled__c": false,
    "agf__Catchup_Factor__c": null,
    "agf__Child_ID__c": null,
    "agf__Closed_By__c": null,
    "agf__Closed_On__c": null,
    "agf__Closed__c": 0.0,
    "agf__Cloud__c": null,
    "agf__Color__c": null,
    "agf__Column_Rank__c": null,
    "agf__Column__c": null,
    "agf__Comment_Copy__c": null,
    "agf__Complete_By__c": null,
    "agf__Created_By_import__c": "0057Q000000x0FIQAY",
    "agf__Created_On_Date__c": "2021-12-06",
    "agf__Created_On_import__c": "2021-12-06T16:15:02.000+0000",
    "agf__Critical_CRM_Feature__c": false,
    "agf__Customer_Impact__c": null,
    "agf__Customer__c": null,
    "agf__Data_Silo_Test_Affected__c": false,
    "agf__Deal_at_Risk__c": null,
    "agf__Dependencies__c": null,
    "agf__Dependents__c": null,
    "agf__Description__c": null,
    "agf__Detailed__c": false,
    "agf__Details__c": null,
    "agf__Details_and_Steps_to_Reproduce__c": null,
    "agf__Due_Date__c": null,
    "agf__Email2GUS_Addresses__c": null,
    "agf__Email_On_Save_Copy__c": false,
    "agf__Email_On_Save__c": false,
    "agf__Email_Subscription_ID__c": null,
    "agf__Email_Subscription_Queue__c": null,
    "agf__Encoded_Recipients_Txt_Area__c": null,
    "agf__Environment__c": null,
    "agf__Epic_Name__c": null,
    "agf__Epic_Rank__c": null,
    "agf__Epic__c": null,
    "agf__Escalation_Point__c": null,
    "agf__Estimated_Financial_Impact__c": null,
    "agf__Executive_Involved__c": false,
    "agf__External_ID__c": null,
    "agf__Feature_Rank__c": null,
    "agf__Feedback__c": null,
    "agf__Found_In_Build_Copy__c": null,
    "agf__Found_In_Build_Name__c": "st1.0",
    "agf__Found_in_Build__c": "a067Q000000ouagQAA",
    "agf__Frequency_Name__c": "Always",
    "agf__Frequency__c": "a0O7Q000000GxXCUA0",
    "agf__Goal__c": null,
    "agf__Has_Story_Points__c": "No",
    "agf__Help_Status__c": null,
    "agf__Highlight__c": false,
    "agf__Impact_Name__c": "Malfunctioning",
    "agf__Impact__c": "a0S7Q000000IZeCUAW",
    "agf__Is_Ignorable__c": false,
    "agf__Is_Template__c": false,
    "agf__Known_Issue_ID__c": null,
    "agf__Known_Issue_Link__c": "No Known Issue",
    "agf__Known_Issue_Num_Reporting_Customers__c": null,
    "agf__Last_Modified_By_Copy__c": null,
    "agf__Last_Modified_By__c": null,
    "agf__Last_Updated_By__c": "0057Q000000tYVk",
    "agf__Log_Bug_From_Template__c": "<a href=\"/a1F7Q000000TRim/e?clone=1&amp;RecordTypeLogicalName=Bug\" target=\"_self\">Log Bug From Template</a>",
    "agf__Major_Func_Area__c": null,
    "agf__Major_Release__c": null,
    "agf__Minor_Func_Area__c": null,
    "agf__Mobile_Device_OS__c": null,
    "agf__Mobile_Device__c": null,
    "agf__Mobile_Network__c": null,
    "agf__Modules__c": null,
    "agf__Num_Of_Prod_Occ__c": null,
    "agf__Number_of_Cases__c": 0.0,
    "agf__Number_of_Change_Lists__c": 0.0,
    "agf__Number_of_Orgs_affected__c": null,
    "agf__Number_of_Tests_Classes_affected__c": null,
    "agf__Origin__c": null,
    "agf__Other_Recipients__c": null,
    "agf__Out_of_SLA__c": false,
    "agf__Parent_ID__c": null,
    "agf__Perforce_Status__c": null,
    "agf__Preserve_Formatting_Copy__c": false,
    "agf__Preserve_Formatting__c": false,
    "agf__Previous_Comments__c": null,
    "agf__Priority_Default__c": null,
    "agf__Priority_Mapping_Copy__c": null,
    "agf__Priority_Mapping__c": null,
    "agf__Priority_Override_Explanation_Copy__c": null,
    "agf__Priority_Override_Explanation__c": null,
    "agf__Priority_Rank__c": null,
    "agf__Priority__c": "P1",
    "agf__Product_Area__c": null,
    "agf__Product_Child__c": null,
    "agf__Product_Owner__c": null,
    "agf__Product_Tag_Name__c": "Sample Product Tag",
    "agf__Product_Tag__c": "a0c7Q000000kFr9QAE",
    "agf__QA_Engineer__c": null,
    "agf__Readme_Notes__c": null,
    "agf__Record_Type__c": "Bug",
    "agf__Red_Account__c": false,
    "agf__Regressed__c": false,
    "agf__Related_URL_Link__c": "<a href=\" \" target=\"_blank\"> </a>",
    "agf__Related_URL__c": null,
    "agf__Related_Work__c": null,
    "agf__Release__c": null,
    "agf__Request_RD_Mgr_Review__c": false,
    "agf__Resolution__c": null,
    "agf__Resolved_By__c": null,
    "agf__Resolved_On__c": null,
    "agf__Resolved__c": 0.0,
    "agf__Root_Cause_Analysis_2__c": null,
    "agf__S1_App_Build_Number__c": null,
    "agf__Scheduled_Build_Copy__c": null,
    "agf__Scheduled_Build_Name__c": null,
    "agf__Scheduled_Build_Rank__c": null,
    "agf__Scheduled_Build__c": null,
    "agf__Scheduled_On__c": null,
    "agf__Schema__c": false,
    "agf__Scrum_Team_Last_Modified__c": "2021-12-06T16:15:02.000+0000",
    "agf__Scrum_Team_Name__c": "Sample",
    "agf__Scrum_Team__c": "a0p7Q000000gBQIQA2",
    "agf__Scrumforce_ID__c": null,
    "agf__Security__c": false,
    "agf__Senior_Management_POC__c": null,
    "agf__Severity_Copy__c": null,
    "agf__Severity_Level__c": null,
    "agf__Solution_Overview__c": null,
    "agf__Sprint_Name__c": "2021.12a-Sample Team",
    "agf__Sprint_Rank__c": null,
    "agf__Sprint_Timeframe__c": "2021.12a",
    "agf__Sprint__c": "a0u7Q000000PIBOQA4",
    "agf__Status__c": "New",
    "agf__Story_Points__c": null,
    "agf__Story_Status__c": "Open",
    "agf__Subject__c": "GPS locations rendering incorrectly",
    "agf__Subscribe_to_Work__c": false,
    "agf__System_Test_Engineer__c": null,
    "agf__Target_Build__c": null,
    "agf__Tech_Writer__c": null,
    "agf__Template_Description__c": null,
    "agf__Template_Name__c": null,
    "agf__Test_Failure_Status__c": null,
    "agf__Test_Plan__c": null,
    "agf__Theme_Rank__c": null,
    "agf__Theme__c": null,
    "agf__Total_Age_When_Closed__c": 0.0,
    "agf__Trust_Rank__c": null,
    "agf__Type_Value__c": 0.0,
    "agf__Type__c": "Bug",
    "agf__UE_Engineer__c": null,
    "agf__Use_Prioritizer__c": true,
    "agf__User_Profile_of_the_Creator__c": null,
    "agf__User_Type__c": null,
    "agf__Was_Ever_Returned_to_Support__c": false,
    "agf__WorkId_and_Subject__c": "W-000000: GPS locations rendering incorrectly",
    "agf__ftest__c": null,
    "agf__of_Test_Failures__c": null,
    "agf__visual_link_num_of_Test_Failures__c": "<a href=\"https://build2.soma.salesforce.com/luna2/bughandlerreport.faces?workid=W-000000\" target=\"new\">0</a>",
    "agf__Completed_Hours__c": 0.0,
    "agf__Number_of_SLA_Violations__c": 0.0
    }
    ```
</details>

[the remark this answers is not in the document]
Comment (cjwooo) [reply destinations/faros-destination/src/converters/agileaccelerator/works.ts:111]: Let's make that list configurable in the destination spec.

[the remark this answers is not in the document]
Comment (Roma-Kyrnis) [reply destinations/faros-destination/src/converters/agileaccelerator/works.ts:111]: Ok, great idea

**Comment by tovbinm** [comment]
Thank you!!

**Pull request merged**